tools/load-generator/Dockerfile

```diff
@@ -1,10 +1,14 @@
-FROM python:3
+FROM golang:1.20-alpine
 
-COPY ./requirements.txt /usr/src/app/
-RUN pip install -r /usr/src/app/requirements.txt
+WORKDIR /usr/src/app
 
-COPY ./*.py /usr/src/app/
+COPY go.mod go.sum ./
+RUN go mod download
 
-WORKDIR /usr/src/app
+COPY . .
+
+RUN go build -o load-generator .
```

Comment:
We don't do builds in Docker here. What you want to do is add the new tool to the top level `.promu.yml`. Take a look at `tools/fake-webserver/Dockerfile` for an example.